dev/source/docs/companion-computers.rst

```diff
@@ -5,24 +5,29 @@ Companion Computers
 ===================
 
 Companion Computers can be used to interface and communicate with
-ArduPilot on a flight controller using the MAVLink protocol.  By doing
+ArduPilot on a flight controller using the MAVLink protocol. By doing
 this your companion computer gets all the MAVLink data produced by the
 autopilot (including GPS data) and can use it to make intelligent
 decisions during flight. For example, "take a photo when the vehicle is
-at these GPS co-ordinates".
+at these GPS co-ordinates", gather and preprocess information from advanced
+sensors or actuate on lights, auxiliary servos or any other interfaces.
```

Comment:
this change needs some work.  Companion computers are increasingly used for "Pilot" functions like collision avoidance and navigation